frontend/src/layouts/explorer/gemBrowser/ComponentLayout.vue

```diff
@@ -30,7 +30,7 @@
               <a href="/api/v2" target="_blank">API</a></p>
           </div>
           <slot v-if="isMetabolite" name="chebi" />
-          <div class="column is-3-widescreen is-3-desktop is-half-tablet has-text-centered">
+          <div class="column is-3-widescreen is-3-desktop is-half-tablet has-text-centered" style="min-width: 350px;">
```

Comment:
Good idea. Commit 0cf2139b7f27ac5b734017e34b220c10b08c1590 incorporates this. but I used is-6-tablet to avoid names with more standard length(example Endoplasmic reticulum or Mitochondria) to get like 1-2 letters broken off when screen is a bit more narrow than tablet

Reply:
Good thinking!